### ISSUE TITLE
Results in Bayesian Optimization

### ISSUE THREAD

**Issue by loukasilias** [muhanzhang/D-VAE#5]
I have run the expirements with DVAE and in bayesian optimization i get the following results in best_arc_scores.txt :
![118801345_672853366922853_1482046152836433382_n](https://user-images.githubusercontent.com/66082450/92098590-1e011280-ede2-11ea-95c4-0634d2f1bd19.png)

Is this metric accuracy? Why is it nearly 10% ?
Thank you in advance.

**Comment by muhanzhang** [comment]
Yes, the metric is accuracy. But the numbers shown are too low, which seems to be from an untrained model (so predicts each of the ten classes of CIFAR10 randomly). Did you download the [pretrained ENAS model](https://www.dropbox.com/sh/h5q9g784uf41xhi/AADZaGvYqHucoQ373U17J_pPa?dl=0) to "software/enas/" following README?

**Comment by loukasilias** [comment]
Thanks for your reply. You are right, i forgot to move the pretrained model to the remote environment. I see now that i get nearly 75% accuracy. Have you included any code in order to train fully the architectures as you mention in your paper, or have you just the weight sharing accuracy? Thanks in advance.

**Comment by muhanzhang** [comment]
Yes. You can copy your found architecture to [fully_train_ENAS.py](https://github.com/muhanzhang/D-VAE/blob/master/bayesian_optimization/fully_train_ENAS.py) and run this file.
